src/util/ui_message.cpp

```diff
@@ -40,7 +40,7 @@ ui_message_handlert::ui_message_handlert(
     break;
 
   case uit::XML_UI:
-    out << "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+    out << "<?xml version=\"1.1\" encoding=\"UTF-8\"?>"
```

Comment:
Let's please not do this for XML 1.1 is not widely supported. XML output is meant for machine consumption, but popular libraries refuse to implement support for it (https://github.com/libexpat/libexpat/issues/378, https://mail.gnome.org/archives/xml/2010-July/msg00017.html).